contracts/src/v0.8/dev/automation/2_1/KeeperRegistry2_1.sol

```diff
@@ -5,36 +5,20 @@ import "../../../vendor/openzeppelin-solidity/v4.7.3/contracts/proxy/Proxy.sol";
 import "../../../vendor/openzeppelin-solidity/v4.7.3/contracts/utils/structs/EnumerableSet.sol";
 import "../../../vendor/openzeppelin-solidity/v4.7.3/contracts/utils/Address.sol";
 import "./KeeperRegistryBase2_1.sol";
-import {AutomationRegistryExecutableInterface, UpkeepInfo} from "../../interfaces/automation/2_1/AutomationRegistryInterface2_1.sol";
-import "../../../interfaces/automation/MigratableKeeperRegistryInterface.sol";
-import "../../../interfaces/automation/MigratableKeeperRegistryInterfaceV2.sol";
+import "./KeeperRegistryLogicA2_1.sol";
+import "./Chainable.sol";
+import {AutomationRegistryExecutableInterface, UpkeepInfo} from "./interfaces/AutomationRegistryInterface2_1.sol";
 import "../../../interfaces/ERC677ReceiverInterface.sol";
 import "../../../OCR2Abstract.sol";
 
-/**
- _.  _|_ _ ._ _  _._|_o _ ._  o _  _    ._  _| _  __|_o._
-(_||_||_(_)| | |(_| |_|(_)| | |_> (_)|_||  (_|(/__> |_|| |\/
-                                                          /
- */
 /**
  * @notice Registry for adding work for Chainlink Keepers to perform on client
  * contracts. Clients must support the Upkeep interface.
  */
-contract KeeperRegistry2_1 is
-  KeeperRegistryBase2_1,
-  Proxy,
-  OCR2Abstract,
-  AutomationRegistryExecutableInterface,
-  MigratableKeeperRegistryInterface,
-  MigratableKeeperRegistryInterfaceV2,
-  ERC677ReceiverInterface
-{
+contract KeeperRegistry2_1 is KeeperRegistryBase2_1, OCR2Abstract, Chainable, ERC677ReceiverInterface {
```

Comment:
Migratable interfaces enforced on LogicA, `AutomationRegistryExecutableInterface` removed entirely (not sure we even need it anymore)